lib/Doctrine/Common/Proxy/ProxyGenerator.php

```diff
@@ -1141,6 +1150,7 @@ private function formatType(
 
         if ($type->allowsNull()
             && (null === $parameter || ! $parameter->isDefaultValueAvailable() || null !== $parameter->getDefaultValue())
+            && $name !== 'mixed'
```

Comment:
mixed returns true in `allowsNull` but if you use it with `?mixed` it throws a fatal error, so this adds a special case.

Reply:
Is that a bug in PHP?

Reply:
No why? allowsNul() is right about true, maybe `?mixed` would not have been a big problem, but its right mixed already contains null, so its redundant.

Reply:
Oh right, indeed.